packages/transition-backend/src/services/nodes/__tests__/NodeCollectionUtils.test.ts

```diff
@@ -9,10 +9,13 @@ import events from 'events';
 import { v4 as uuidV4 } from 'uuid';
 import { NodeAttributes } from 'transition-common/lib/services/nodes/Node';
 import NodeCollection from 'transition-common/lib/services/nodes/NodeCollection';
-import { saveAndUpdateAllNodes } from '../NodeCollectionUtils';
+import { saveAndUpdateAllNodes, saveAllNodesToCache } from '../NodeCollectionUtils';
 import { objectToCache } from '../../../models/capnpCache/transitNodes.cache.queries';
 import { getTransferableNodes } from '../TransferableNodeUtils';
+import transferableNodesDbQueries from '../../../models/db/transitNodeTransferable.db.queries';
 
+// Mock all DB queries about transferable nodes
+jest.mock('../../../models/db/transitNodeTransferable.db.queries');
```

Comment:
You might want to make some check on these functions, with some returning some transferrable nodes, and you can then make sure the transferable nodes are also sent to the save method.